app/javascript/admin/controllers/creator_settings_controller.js

```diff
@@ -67,4 +67,53 @@ export class CreatorSettingsController extends Controller {
 
     reader.readAsDataURL(firstFile);
   }
+
+  /**
+   * Updates ths branding/colors on the Creator Settings Page.
+   *
+   * @param {Event} event
+   */
+  updateBranding(event) {
+    const color = event.target.value;
+
+    document.documentElement.style.setProperty('--accent-brand', color);
+
+    // responsible for the hover effect over the button.
+    // We need to recalculate this in javascript as it's currently being calculated in ruby
+    document.documentElement.style.setProperty(
```

Comment:
For the hover effect on buttons we need to calculate a new `--accent-brand-darker` which decreases the brightness by 85%.